src/MeetingsSDKAdapter/controls/ExitControl.test.js

```diff
@@ -0,0 +1,37 @@
+import {first} from 'rxjs/operators';
+import {meetingID, createTestMeetingsSDKAdapter} from '../testHelper';
+
+describe('Exit Control', () => {
+  let meetingsSDKAdapter;
+
+  beforeEach(() => {
+    meetingsSDKAdapter = createTestMeetingsSDKAdapter();
+  });
+
+  afterEach(() => {
+    meetingsSDKAdapter = null;
+  });
+
+  describe('display()', () => {
+    test('returns the display data in a proper shape', (done) => {
+      meetingsSDKAdapter.meetingControls['leave-meeting'].display().pipe(first()).subscribe((dataDisplay) => {
+        expect(dataDisplay).toMatchObject({
+          ID: 'leave-meeting',
+          icon: 'cancel_28',
+          tooltip: 'Leave',
+          state: 'active',
+        });
+        done();
+      });
+    });
+  });
+
+  describe('action()', () => {
+    test('calls leaveMeeting() SDK adapter method', async () => {
+      meetingsSDKAdapter.leaveMeeting = jest.fn();
+      await meetingsSDKAdapter.meetingControls['leave-meeting'].action(meetingID);
+      expect(meetingsSDKAdapter.leaveMeeting).toHaveBeenCalledTimes(1);
```

Comment:
no need for this, `sdk.leaveMeeting` is idempotent (even if it's called multiple times, the effect is the same)